ansible/playbook.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Configure machine and launch app
+  hosts: managed
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Enter trello api key immediately
+    - name: trello_api_token
+      prompt: Enter trello api token immediately
+
+  tasks:
+  - name: Install git and python
+    become: yes
+    ansible.builtin.yum:
+      name:
+        - git
+        - python3
+  - name: Ensure poetry installed
+    ansible.builtin.shell:
+      cmd: install_poetry.sh
+      creates: .local/share/pypoetry/venv/bin/poetry
+  - name: Ensure project directory exists with correct permissions
+    become: yes
+    ansible.builtin.file:
+      path: /opt/todo_app
+      state: directory
+      owner: ec2-user
+  - name: Checkout latest app code
+    ansible.builtin.git:
+      repo: 'https://github.com/arfacamble/DevOps-Course-Starter.git'
+      dest: /opt/todo_app
+      version: master
+  - name: Poetry install
+    ansible.builtin.shell:
+      cmd: (cd /opt/todo_app && poetry=$(which poetry) && $poetry install)
```

Comment:
Ooh interesting solution to assign poetry. Does this actually help though? If `which poetry` can find poetry, then isn't that just using the same executable as a bare `poetry install` would?
I can believe I could be wrong, but can't see why this helps!